avocado/core/multiplexer.py

```diff
@@ -20,66 +20,12 @@
 """
 
 import copy
-import collections
-import itertools
 import logging
 import re
 
 from . import tree
 
 
-class MuxTree(object):
-
-    """
-    Object representing part of the tree from the root to leaves or another
-    multiplex domain. Recursively it creates multiplexed variants of the full
-    tree.
-    """
-
-    def __init__(self, root):
-        """
-        :param root: Root of this tree slice
-        """
-        self.root = root
```

Comment:
This is pretty minor, and it's also something core, so users wouldn't be affect.  It just serves as a best practice reminder.